modules/virt-runbook-virtualmachinecrcerrors.adoc

```diff
@@ -27,41 +27,45 @@ severe performance degradation.
 [id="diagnosis-virtualmachinecrcerrors"]
 == Diagnosis
 
-. Get the volume name from a virtual machine:
+. Navigate to *Observe* -> *Metrics* in the web console.
+. Obtain a list of VirtualMachines with incorrectly configured storage classes
```

Comment:
The rest of this assembly (and book overall) seems to use "virtual machines (VMs)". Can this module do this same? Otherwise, "VirtualMachines" seems more like an API [object reference](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#object-references) (i.e., would need backticks + descriptor).